packages/react-composites/src/composites/CallComposite/components/MediaGallery.tsx

```diff
@@ -60,11 +65,58 @@ export const MediaGallery = (props: MediaGalleryProps): JSX.Element => {
     };
   }, [cameraSwitcherCallback, cameraSwitcherCameras]);
 
+  const [augmentedParticipants, setAugmentedParticipants] = useState<VideoGalleryRemoteParticipant[] | undefined>(
+    undefined
+  );
+
+  // Async function to retrieve custom data model information to replace display name in remote tiles with camera on.
+  const fetchAvatarPersonaDataAsync = memoizeFunction(
+    async (
+      onFetchAvatarPersonaData: AvatarPersonaDataCallback,
+      participants: VideoGalleryRemoteParticipant[]
+    ): Promise<VideoGalleryRemoteParticipant[] | undefined> => {
+      if (props.onFetchAvatarPersonaData) {
```

Comment:
Why are we checking `props.onFetchAvatarPersonaData` here when `onFetchAvatarPersonaData` is a function argument?

Reply:
good point 😅 will update